docs/superpowers/specs/2026-06-04-release-contributor-acknowledgements-design.md

```
@@ -0,0 +1,129 @@
+
+**Extraction logic:**
+
+1. Get the date of `prev_tag` via `git log -1 --format=%aI <prev_tag>`
+2. Query merged PRs: `gh pr list --state merged --search "merged:>=<date>" --json number,title,author,labels`
+3. Filter out excluded authors (bots and maintainer)
+4. Deduplicate by author (one line per contributor, even with multiple PRs)
+5. Format each contributor as: `- @author: description (#123)`
+
+**Exclusion list** (variable at top of script for easy editing):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)